lib/rules/web/admin_console/4.4/common_ui_elements.xyaml

```diff
@@ -55,7 +55,7 @@ select_resource_item:
 click_on_resource_name:
   element:
     selector:
-      xpath: //a[contains(@class,'co-resource-item__resource-name') and text()='<item>']
+      xpath: //span[contains(@class,'co-resource-item__resource-name') and text()='<item>']
```

Comment:
Maybe `//*[contains(@class,'co-resource-item__resource-name') and text()='<item>']` ? It looks it changes very often